jsonp-platform-tck/pom.xml

```
@@ -0,0 +1,351 @@
+                                </goals>
+                                <phase>process-test-resources</phase>
+                                <configuration>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>jakarta.tck.arquillian</groupId>
+                                            <artifactId>arquillian-protocol-lib</artifactId>
+                                            <type>jar</type>
+                                            <overWrite>true</overWrite>
+                                            <outputDirectory>${project.build.directory}${file.separator}protocol</outputDirectory>
+                                            <destFileName>protocol.jar</destFileName>
+                                        </artifactItem>
+                                    </artifactItems>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <plugin>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>patch-tests-appclient</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <additionalClasspathElements>
+                                        <additionalClasspathElement>${project.build.directory}${file.separator}${payara.home}${file.separator}glassfish${file.separator}modules${file.separator}parsson.jar</additionalClasspathElement>
+                                    </additionalClasspathElements>
+                                    <includes>
+                                        <include>
+                                            com/sun/ts/tests/jsonp/api/patchtests/*Test*.java</include>
+                                    </includes>
+                                    <!-- Select the @Tag("tck-appclient") tests -->
+                                    <groups>tck-appclient</groups>
+                                    <dependenciesToScan>jakarta.tck:jsonp-platform-tck</dependenciesToScan>
+                                    <systemPropertyVariables>
+                                        <payara.home>${payara.home}</payara.home>
+                                        <junit.log.traceflag>true</junit.log.traceflag>
+                                        <harness.log.traceflag>true</harness.log.traceflag>
+                                        <cts.harness.debug>true</cts.harness.debug>
+                                        <java.io.tmpdir>/}tmp</java.io.tmpdir>
```

Presumably an incorrect substitution?
```suggestion
                                        <java.io.tmpdir>${java.io.tmpdir}</java.io.tmpdir>
```

---

yeah, it seems that my TAB key went off the grid 🔢